internal/workflow_test.go

```diff
@@ -56,6 +56,8 @@ func TestGetChildWorkflowOptions(t *testing.T) {
 			"foo": "bar",
 		},
 		ParentClosePolicy: enums.PARENT_CLOSE_POLICY_REQUEST_CANCEL,
+		// TODO(bergundy): test the defaults here with caller and destination same task queue / different task queue
```

Comment:
Can you add tests for the defaults for activity, child, and CAN please?
Can also be done in the features repo.

In any case let's get rid of this TODO before merging.

Reply:
This can't really be done in a super sane way in the unit tests that I can find easily, but, I've added a test for the helper function I made. Features repo can do the e2e validation

Reply:
SGTM